pages/getting-started/install-memgraph/kubernetes.mdx

```diff
@@ -111,8 +113,10 @@ their default values.
 | `persistentVolumeClaim.logStorageClassName` | Storage class name for the persistent volume claim for logs                                         | `""`                                    |
 | `persistentVolumeClaim.logStorageSize`      | Size of the persistent volume claim for logs                                                        | `1Gi`                                   |
 | `memgraphConfig`                            | List of strings defining Memgraph configuration settings                                            | `["--also-log-to-stderr=true"]`         |
-| `memgraphUser`                              | User for the Memgraph database                                                                      | `""`                                    |
-| `memgraphPassword`                          | Password for the Memgraph database                                                                  | `""`                                    |
+| `secrets.enabled`                           | Enable the use of Kubernetes secrets for Memgraph credentials                                       | `false`                                 |
+| `secrets.name`                              | The name of the Kubernetes secret containing Memgraph credentials                                   | `memgraph-secrets`                      |
+| `secrets.userKey`                           | The key in the Kubernetes secret for the Memgraph user                                              | `MEMGRAPH_USER`                         |
+| `secrets.passwordKey`                       | The key in the Kubernetes secret for the Memgraph password                                          | `MEMGRAPH_PASSWORD`                     |
```

Comment:
What is a "key" in this context? Why is the string "key" present in the parameter name, but missing in the environment variable - that's confusing and inconsistent.

Reply:
Here is the problem: this is not an `env` variable, this is a secret key, that you go and set some value to, and then that value is passed to the actuall env. 

Like this: 
```
            - name: MEMGRAPH_USER
              valueFrom:
                secretKeyRef:
                  name: {{ .Values.secrets.name }}
                  key: {{ .Values.secrets.userKey }}
            - name: MEMGRAPH_PASSWORD
              valueFrom:
                secretKeyRef:
                  name: {{ .Values.secrets.name }}
                  key: {{ .Values.secrets.passwordKey }}
```

You set the key values like this: 
```kubectl create secret generic memgraph-secrets --from-literal=MEMGRAPH_USER=myuser --from-literal=MEMGRAPH_PASSWORD=mypassword```

So the value is implicitly read from the key and set to the `MEMGRAPH_USER` and `MEMGRAPH_PASSWORD`. In this case, the key share the same naming as the env variables. But I can agree it is a bit confusing. 

I will rename this to a USER and PASSWORD, so there is a distinction form envs, but you still have the key step.